atlas-content-modeler/tutorial/models/fields/relationship/cardinality.mdx

```diff
@@ -0,0 +1,43 @@
+---
+slug: /atlas-content-modeler/tutorial/models/fields/relationship/cardinality
+title: Atlas Content Modeler Fields - Relationship - Cardinality
+description: Learn about Cardinality in Relationship Fields in Atlas Content Modeler.
+---
+
+**Note**: These tutorials assume that you have already [installed the plugin](../../../../introduction.mdx).
+
+## Relationship Field (Beta) - Cardinality
+
+Cardinality provides a way to setup model relationship restrictions. It's a connection or reference between one or more entries of a model with one or more entries of another model. There are 4 types: One-to-One, One-to-Many, Many-to-One, and Many-to-Many. Cardinality types that only allow one selection use a Radio button in the UI. The types that allow many use Checkboxes in the UI.
+
+### Steps to Use the Relationship Field Cardinality
+
+---
+
+1. After navigating to a model, click the + button or if it's the first field, click on the Relationship option
+
+   ![Atlas Content Modeler sync](../../../../images/field_options.png)
+
+1. Choose the Relationship Field type
+
+1. Fill in the form fields to fit your needs
+
+   ![Atlas Content Modeler sync](../../../../images/relationships_cardinality.png)
+
+1. Under Connections, choose a Cardinality option:
+
+   ![Atlas Content Modeler sync](../../../../images/cardinality.png)
+
+   1. One-to-One: One of this model type can have one related item
+   1. One-to-Many: One of this model type can have many related items
+   1. Many-to-One: Many of this model type can have one related items
+   1. Many-to-Many: Many of this model type can have many related items
```

Comment:
Thoughts on changing "of this model type" to "{item/items} of this model type"? It makes it more explicit I think. 

Examples:
One-to-One: Each item of this model type can have one related item

Many-to-One: Many items of this model type can have one related item

Note: also that the third list item ends with plural "items" and should be singular "item"